index.js

```diff
@@ -1,12 +1,95 @@
 const express = require('express');
+const fs = require('fs');
 
 const PORT = 3000;
 const app = express();
 
 app.use(express.json());
 
+app.use(((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Headers', '*');
+
+  if (req.method === 'OPTIONS') {
+    res.header('Access-Control-Allow-Methods', 'PUT, POST, PATCH, DELETE, GET');
+    return res.status(200).json({});
+  }
+  next();
+}))
+
 app.get('/api', ((req, res) => {
   res.send('Hello from WHAT mock server!');
 }));
 
+app.get('/api/courses', ((req, res) => {
+  fs.readFile('./mocks/courses.json', ((err, data) => {
+    if (err) {
+      req.status(500).send('Server error occured');
```

Comment:
How do you know that you should send 500 response status?
Maybe you should `throw err`?